src/plugins/cosmos/components/modals/Staking/views/Unstake.tsx

```diff
@@ -0,0 +1,237 @@
+      setValue(Field.CryptoAmount, cryptoAmount.toString(), { shouldValidate: true })
+    } else {
+      setValue(Field.FiatAmount, fiat.toString(), { shouldValidate: true })
+      setValue(Field.CryptoAmount, cryptoAmount.toString(), { shouldValidate: true })
+    }
+    setPercent(_percent)
+  }
+
+  const handleInputChange = (value: string) => {
+    setPercent(null)
+    if (activeField === InputType.Crypto) {
+      const fiat = bnOrZero(value).times(marketData.price)
+      setValue(Field.FiatAmount, fiat.toString(), { shouldValidate: true })
+    } else {
+      const crypto = bnOrZero(value).div(marketData.price)
+      setValue(Field.CryptoAmount, crypto.toString(), { shouldValidate: true })
+    }
+  }
+
+  const handleInputToggle = () => {
+    const field = activeField === InputType.Crypto ? InputType.Fiat : InputType.Crypto
+    if (fieldError) {
+      // Toggles an existing error to the other field if present
+      clearErrors(fiatError ? Field.FiatAmount : Field.CryptoAmount)
+      setError(fiatError ? Field.CryptoAmount : Field.FiatAmount, {
+        message: 'common.insufficientFunds'
+      })
+    }
+    setActiveField(field)
+  }
+  // TODO: wire me up, parentheses are nice but let's get asset name from selectAssetNameById instead of this
+  const asset = (_ => ({
+    name: 'Osmo',
+    symbol: 'OSMO'
+  }))(assetId) as Asset
+  return (
+    <AnimatePresence exitBeforeEnter initial={false}>
+      <Box
+        as='form'
+        pt='22px'
+        pb='15px'
+        px='24px'
+        maxWidth='lg'
+        width='full'
+        onSubmit={handleSubmit(onSubmit)}
+      >
+        <Flex
+          direction='column'
+          maxWidth='595px'
+          minHeight='380px'
+          alignItems='center'
+          justifyContent='space-between'
+        >
+          <CosmosActionButtons
+            asset={asset}
+            activeAction={StakingAction.Unstake}
+            width='100%'
+            px='6px'
+            py='6px'
+            bgColor='gray.850'
+            borderRadius='12px'
+          />
+          <Box>
+            <CText color='gray.500' my='25px'>
+              {translate('staking.itWillTake')}
+              <span> </span>
+              <Box as='span' fontWeight='semibold' color='white'>
+                14 {translate('common.days')}
+              </Box>
+              <span> </span>
+              {translate('staking.toUnlock')}
+            </CText>
+          </Box>
+          <Flex width='100%' mb='6px' justifyContent='space-between' alignItems='center'>
+            <Text
+              lineHeight={1}
+              color='gray.500'
+              translation={['staking.assetStakingBalance', { assetName: asset.name }]}
+            />
+            <Amount.Crypto value={cryptoAmountStaked || ''} symbol={asset.symbol} />
+          </Flex>
+          <Divider />
+          <FormControl>
+            <AmountToStake
+              width='100%'
```

Comment:
```suggestion
```

You should test without this one, looks useless as `FormControl` is just a div with `width: 100%` and `AmountToStake` is a flexbox, it's probably naturally 100%

If not, then just resolve it